core/variant_parser.cpp

```diff
@@ -1414,7 +1414,7 @@ Error VariantParser::parse_value(Token& token,Variant &value,Stream *p_stream,in
 			return OK;
 		} else if (id=="img") {  // compatibility with godot.cfg
 
-			Token token;
+			Token token;	// no need for this declaration? the first argument in line 509 is a Token& token.
```

Comment:
I guess the idea is that it shadows the other `token` without modifying its data (while if it modified the `Token&` reference it would change its data). Whether that's necessary or not, I don't know :)